alcs-frontend/src/app/features/application/application.component.html

```diff
@@ -9,11 +9,14 @@
       [showStatus]="true"
       [submissionStatusService]="applicationStatusService"
     ></app-details-header>
-    <div class="content">
+    <div class="content" *ngIf="application">
       <div *ngIf="showSubmittedToAlcMenuItems" class="nav">
         <div *ngFor="let route of childRoutes" class="nav-link">
           <div
-            *ngIf="route.portalOnly ? isApplicantSubmission : true"
+            *ngIf="
+              (route.portalOnly ? isApplicantSubmission : true) &&
```

Comment:
I'm pretty certain this is equivalent to `!route.portalOnly || isApplicantSubmission`, which would be clearer (same for following line).

Reply:
Tested and looks good, Done!